ci/test_python.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -euo pipefail
+
+rapids-logger "Create test conda environment"
+. /opt/conda/etc/profile.d/conda.sh
+
+rapids-dependency-file-generator \
+  --generate conda \
+  --file_key test \
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" > env.yaml
+
+rapids-mamba-retry env create --force -f env.yaml -n test
+conda activate test
+
+rapids-print-env
+
+rapids-logger "Downloading artifacts from previous jobs"
+CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
+PYTHON_CHANNEL=$(rapids-download-conda-from-s3 python)
+
+rapids-mamba-retry install \
+  -c "${CPP_CHANNEL}" \
+  -c "${PYTHON_CHANNEL}" \
+  rmm librmm
+
+TESTRESULTS_DIR="${GITHUB_WORKSPACE}/test-results"
```

Comment:
```suggestion
TESTRESULTS_DIR="${PWD}/test-results"
```
Will `PWD` work here? I think it's good to rely on CI-specific variables as little as possible.